lib/query/stored.js

```diff
@@ -45,8 +44,53 @@ const deleteStoredQuery = (conn, storedQuery, params) => {
   }).then(httpBody);
 };
 
+const replace = (conn, storedQuery, params) =>
+  deleteStoredQuery(conn, storedQuery.name).then(deleteResponse => {
+    //  Update creates when the query did not already exist
+    if (deleteResponse.status === 404 || deleteResponse.ok) {
```

Comment:
I wanted to distinguish between 404 failures and non-404 failures. If it's a 404 I assume it means the query didn't exist and we can continue. Otherwise who knows what's happening.

Reply:
That makes sense. 👍